hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4137,6 +4137,14 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.short.circuit.num</name>
+  <value>1</value>
+  <description>
+    Number of short-circuit caches.
```

Comment:
Please update with the allowed range here. ( 1 <= num <= 5)

Reply:
Done